group_vars/all.yaml

```diff
@@ -33,6 +33,8 @@ jupyterhub:
   base_url: "/"
   hub_environment: jupyterhub
   lab_environment: jupyterlab
+  spawner:
+    start_timeout: 10  # seconds
```

Comment:
makes the spawner timeout configurable by the user

Reply:
Is there a reason the timeout is so low? Could we use a default of 30 or 60 seconds? I think the default is 5 minutes with jupyterhub normally.